### PR TITLE
Use rust default toolchain for ci tests

### PR DESCRIPTION
**Pull request by Chralt98**

### CODE REVIEW

File: rust-toolchain [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [toolchain]
 channel = "nightly-2022-09-24"
-components = ["clippy", "rustfmt"]
+components = ["clippy", "rustfmt", "llvm-tools-preview"]
```
Comment (maltekliemann) [inline rust-toolchain:3]: What's this for?

Comment (Chralt98) [reply rust-toolchain:3]: It's for the test code coverage.

Comment (Chralt98) [reply rust-toolchain:3]: https://github.com/zeitgeistpm/zeitgeist/blob/63d5fda89e000e6a75ccc5771860710cd859b07c/.github/workflows/rust.yml#L71-L100

Does this help?

Comment (Chralt98) [reply rust-toolchain:3]: https://blog.rng0.io/how-to-do-code-coverage-in-rust

Comment (maltekliemann) [reply rust-toolchain:3]: Ah, yeah, thanks, I just can't read.

Comment (Chralt98) [reply rust-toolchain:3]: https://github.com/zeitgeistpm/zeitgeist/blob/1f7ea6192eb9ad5921a99ad959e14d9065d8ca09/scripts/tests/aux-functions.sh#L20

**Review by sea212 — APPROVED** [review]

**Pull request merged**